ctru-sys/build.rs

```diff
@@ -30,9 +45,97 @@ fn main() {
         }
         Err(err) => println!("cargo:warning=failed to check libctru version: {err}"),
     }
+
+    let gcc_version = get_gcc_version(PathBuf::from(&devkitarm).join("bin/arm-none-eabi-gcc"));
+
+    let include_path = PathBuf::from_iter([devkitpro.as_str(), "libctru", "include"]);
+    let ctru_header = include_path.join("3ds.h");
+
+    let sysroot = Path::new(&devkitarm).join("arm-none-eabi");
+    let system_include = sysroot.join("include");
+    let gcc_include = PathBuf::from(format!(
+        "{devkitarm}/lib/gcc/arm-none-eabi/{gcc_version}/include"
+    ));
+    let errno_header = system_include.join("errno.h");
+
+    // Build libctru bindings
+    let bindings = Builder::default()
+        .header(ctru_header.to_str().unwrap())
+        .header(errno_header.to_str().unwrap())
```

Comment:
Surprised that this header needs to be included explicitly, but if it's not working without then :shrug:

Reply:
As the [comment](https://github.com/rust3ds/ctru-rs/pull/110#issuecomment-1507139787) you made to support this change, the reason why we need to explicitly include it is because it's not included neither by the C standard library nor by `libctru`.

Reply:
Lol — forgot about that, thanks for the reminder.